static/app/views/replays/detail/network/index.tsx

```diff
@@ -91,8 +91,16 @@ function NetworkList({replayRecord, networkSpans}: Props) {
     return (
       <Fragment key={index}>
         <Item>{<StatusPlaceHolder height="20px" />}</Item>
-        <Item color={theme.gray400}>
-          <Text>{network.description || <Placeholder height="24px" />}</Text>
+        <Item color="gray400">
+          <Tooltip
+            title={network.description}
+            isHoverable
+            overlayStyle={{
+              maxWidth: '500px !important',
+            }}
+          >
+            <Text>{network.description || <Placeholder height="24px" />}</Text>
```

Comment:
When would this case exist where `network` is non-empty but `description` is empty? Should that be considered "loading" or is it just an empty description/bad data?

Reply:
I think it would be an empty description/bad data - as its type is optional: 
![image](https://user-images.githubusercontent.com/39612839/182397540-dfee1581-c238-46af-813d-799de9235b91.png)

Reply:
Uhmm I see what you mean, maybe we should replace the placeholder by something like this? 

![image](https://user-images.githubusercontent.com/39612839/182401625-a3cf143c-1983-46bc-a374-a05e43d0eda1.png)

Reply:
IMO if a column just doesn't have data but the row is loaded it shouldn't use a placeholder that is used for loading. Just leave it blank or maybe some default text like _(No description)_

Reply:
This is the result!

![image](https://user-images.githubusercontent.com/39612839/182647019-5f679079-22a7-4713-b9d4-03a3d732dfeb.png)

Reply:
I'm just giving my thoughts maybe Ryan or Billy have a better idea or opinion on it. I think if we do keep something like that I would def dim the text at the least maybe make it smaller since it's a not important piece of info in the table row

Reply:
Yeah imo this looks good, it definitely better than what I had before 
![image](https://user-images.githubusercontent.com/39612839/182650796-fb304a37-66dd-45bb-8bee-9c1b89d14297.png)